polyfills/smoothscroll/detect.js

```diff
@@ -0,0 +1 @@
+'scrollBehavior' in d.documentElement.style
```

Comment:
Is d meant to be document?
```suggestion
'scrollBehavior' in document.documentElement.style
```

Reply:
Whoops, yes.